README.adoc

```diff
@@ -211,63 +270,66 @@ You will see a login form since form authentication is implemented and configure
 
 Notice that when you sign in as Bob or Carl, the browser redirects to the `admin` page and you can view their names and roles.
 When you sign in as Alice, you can only view Alice's name.
-When you sign in as Dave, you'll be blocked and will see a `Error 403: Authorization failed` message because Dave doesn't have a role supported by the application.
+When you sign in as Dave, you'll be blocked and will see a `Error 403: Authorization failed` message 
+because Dave doesn't have a role supported by the application.
```

Comment:
Update to:
"When you sign in as Dave, you are blocked and see an `Error 403: Authorization failed` message because Dave doesn't have a role that is supported by the application."